.github/workflows/ci.yml

```diff
@@ -324,15 +324,23 @@ jobs:
     name: "Test examples of production image building"
     runs-on: ${{ fromJson(needs.build-info.outputs.runsOn) }}
     needs: [build-info]
+    defaults:
+      run:
+        working-directory: ./dev/breeze
```

Comment:
I think we have other "runs" here - which use ./scripts/ci  - so at least for now The `working-directory` should only be set fo the "pip install ."  command. Or maybe even better ise `pip install dev/breeze` without changing directory.

 And BTW. You do not need (and it likely will not work) to run `freespace` (this is the name of the entrypoint). No need  to add .py at the end.